tradeit-android-sdk/src/main/java/it/trade/android/sdk/model/TradeItApiClientParcelable.java

```diff
@@ -64,4 +65,8 @@ public TradeItApiClientParcelable createFromParcel(Parcel source) {
             return new TradeItApiClientParcelable[size];
         }
     };
+
+    private static boolean forceTLS12() {
+        return (Build.VERSION.SDK_INT >= 16 && Build.VERSION.SDK_INT < 22);
```

Comment:
It seems the error happens also on some Samsung devices with API 21, so limiting version < 22